spec/scorecardSpec.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+describe('Scorecard', function(){
+
+  var scorecard;
```

Comment:
Our Scorecard tests currently rely on the actual implementation of the Frame class. Can you think of a way to use [mocking](https://github.com/makersacademy/course/blob/master/pills/mocking_in_javascript.md) to isolate the unit tests?